dll/appcompat/apphelp/apphelp.c

```diff
@@ -4,27 +4,38 @@
  * PURPOSE:     apphelp entrypoint / generic interface functions
  * COPYRIGHT:   Copyright 2011 André Hentschel
  *              Copyright 2013 Mislav Blaževic
- *              Copyright 2015-2018 Mark Jansen (mark.jansen@reactos.org)
+ *              Copyright 2015-2019 Mark Jansen (mark.jansen@reactos.org)
  */
 
 #define WIN32_NO_STATUS
 #include "windef.h"
 #include "winbase.h"
-#include "winver.h"
 #include "strsafe.h"
 #include "apphelp.h"
-#include "ndk/rtlfuncs.h"
-#include "ndk/kdtypes.h"
+#include <ndk/rtlfuncs.h>
+#include <ndk/cmfuncs.h>
+#include <ndk/obfuncs.h>
+#include <ndk/kdtypes.h>
 
 
+ACCESS_MASK Wow64QueryFlag(void);
+
+#define INSTALLEDSDB_KEYNAME L"\\Registry\\Machine\\Software\\Microsoft\\Windows NT\\CurrentVersion\\AppCompatFlags\\InstalledSDB"
```

Comment:
Make it UNICODE_STRING directly. Given your usage, you'll save time (a little ;-)).

Reply:
If I don't forget ;)